skrub/datasets/_fetching.py

```diff
@@ -430,6 +432,70 @@ def _fetch_figshare(
             raise URLError("No internet connection or the website is down.")
 
 
+def _fetch_movielens(dataset_id: str, data_directory: Path | None = None) -> dict[str]:
```

Comment:
Without getting into the specifics, you might benefit from having a look at #669 even before it is merged, it would make the adaptation simpler :)

Reply:
I prefer to adapt the dataset fetcher in a subsequent PR to stay lean on this. The risk is having this PR coupled with #669.